src/sentry/search/django/backend.py

```diff
@@ -457,7 +457,7 @@ def _query(self, project, retention_window_start, group_queryset, tags, environm
                     select={
                         'sort_value': get_sort_expression(group_tag_value_queryset.model),
                     },
-                ).values_list('group_id', 'sort_value')
+                ).values_list('group_id', 'sort_value')[:10000]
```

Comment:
This is redundant — this will already be enforced due to the other limit on L431 (that query as limited here can't return > 10,000 results.)

Reply:
This is what I thought, but wasn't 100% sure. I'll just remove it.

Reply:
Removed.